byterun/Makefile.nt

```diff
@@ -33,7 +34,7 @@ ocamlrun$(EXE): libcamlrun.$(A) prims.$(O)
 	         $(call SYSLIB,ws2_32) $(EXTRALIBS))
 
 ocamlrund$(EXE): libcamlrund.$(A) prims.$(O) main.$(O)
-	$(MKEXE) -o ocamlrund$(EXE) $(BYTECCDBGCOMPOPTS) prims.$(O) \
+	$(MKEXE) -o ocamlrund$(EXE) prims.$(O) \
```

Comment:
It makes no sense to include `$(BYTECCDBGCOMPOPTS)` here (unsurprisingly, flexlink doesn't like the extra arguments).